File: openalprwebhookprocessor.client/src/app/settings/enrichers/enricher.ts
import { EnrichmentType } from './enrichmentType';

export class Enricher {
    id: string;
    isEnabled: boolean;
    apiKey: string;
    enrichmentType: EnrichmentType;
    
    constructor(init?: Partial<Enricher>) {
        Object.assign(this, init);
    }
}

File: OpenAlprWebhookProcessor.Server/Settings/Enrichers/Enricher.cs
using OpenAlprWebhookProcessor.Features.LicensePlates.Commands.EnrichPlate;
using System;

namespace OpenAlprWebhookProcessor.Settings.Enrichers
{
    public class Enricher
    {
        public Guid Id { get; set; }

        public bool IsEnabled { get; set; }

        public EnricherType EnricherType { get; set; }

        public string ApiKey { get; set; }

        public EnrichmentType EnrichmentType { get; set; }
    }
}
